### PR TITLE
samplecode refactored

### PR DESCRIPTION
**Pull request by Aneil26**
@nkramer

### CODE REVIEW

**Comment by nkramer** [comment]
What's the empty WebActivatorEx folder?

File: CSharp/Microsoft_Teams_Graph_RESTAPIs_Connect/Microsoft_Teams_Graph_RESTAPIs_Connect/ImportantFiles/GraphService.cs [hunk 1]
```diff
@@ -115,9 +128,92 @@ public class GraphService
...
+            HttpResponseMessage response = await ServiceHelper.SendRequest(new HttpMethod("Patch"), endpoint, accessToken, team);
+            if (!response.IsSuccessStatusCode)
+                throw new Exception(response.ReasonPhrase);
+            return response.ReasonPhrase;
+        }
+
+
+
+
+        public async Task AddMember(string teamId, Member member, string accessToken)
```
Comment (nkramer) [inline CSharp/Microsoft_Teams_Graph_RESTAPIs_Connect/Microsoft_Teams_Graph_RESTAPIs_Connect/ImportantFiles/GraphService.cs]: There's a couple problems here. First, despite what I said earlier it's not actually necessary to turn a upn into an id, you can pass a UPN directly to /groups/{teamId}/members/$ref.

Second, the "Member" class is problematic. There is no such resource type in graph – there's User, and it doesn't have an owner property. I suggest changing AddMember() to take a upn and bool isOwner property.

Comment (nkramer) [reply CSharp/Microsoft_Teams_Graph_RESTAPIs_Connect/Microsoft_Teams_Graph_RESTAPIs_Connect/ImportantFiles/GraphService.cs]: Also, for consistency we should call it groupId not teamId

File: CSharp/Microsoft_Teams_Graph_RESTAPIs_Connect/.gitattributes [hunk 2]
```diff
@@ -0,0 +1,63 @@
+###############################################################################
```
Comment (nkramer) [inline CSharp/Microsoft_Teams_Graph_RESTAPIs_Connect/.gitattributes]: Do we need this file?

File: CSharp/Microsoft_Teams_Graph_RESTAPIs_Connect/.gitignore [hunk 3]
```diff
@@ -0,0 +1,261 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (nkramer) [inline CSharp/Microsoft_Teams_Graph_RESTAPIs_Connect/.gitignore]: How about this file?

File: CSharp/Microsoft_Teams_Graph_RESTAPIs_Connect/GraphAPI.Common/GraphAPI.Common.csproj [hunk 4]
```diff
@@ -0,0 +1,58 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (nkramer) [inline CSharp/Microsoft_Teams_Graph_RESTAPIs_Connect/GraphAPI.Common/GraphAPI.Common.csproj]: Is this part of the final solution or a leftover from previous versions?

**Comment by jasonjoh** [comment]
Closing this as we are closing this repository and will no longer maintain it. The Teams code snippets in this sample are replicated in the [ASP.NET Core Graph Snippets sample](https://github.com/microsoftgraph/aspnet-snippets-sample). We apologize for any inconvenience.

**Pull request closed without merging**